src/view/configuration/personalizationSection.jsx

```diff
@@ -22,24 +23,34 @@ import copyPropertiesWithDefaultFallback from "./utils/copyPropertiesWithDefault
 import FormElementContainer from "../components/formElementContainer";
 import FormikCheckbox from "../components/formikReactSpectrum3/formikCheckbox";
 import BetaBadge from "../components/betaBadge";
+import FormikRadioGroup from "../components/formikReactSpectrum3/formikRadioGroup";
 
 export const bridge = {
   getInstanceDefaults: () => ({
     prehidingStyle: "",
     targetMigrationEnabled: false,
-    personalizationStorageEnabled: false
+    personalizationStorageEnabled: false,
+    autoTrackPropositionInteractions: "always"
   }),
   getInitialInstanceValues: ({ instanceSettings }) => {
     const instanceValues = {};
 
+    if (instanceSettings.autoTrackPropositionInteractions?.AJO) {
```

Comment:
How can a customer enable `autoTrackPropositionInteractions` for Target if they want to in 6 months?

Reply:
They wait for a Web SDK update.